include/libopencm3/stm32/l0/adc.h

```diff
@@ -56,6 +56,24 @@
 /* Calibration Factors */
 #define ADC_CALFACT(adc)	MMIO32((adc) + 0xB4)
 
+/* Register definitions */
+
+#define ADC_SMPR(adc)		ADC_SMPR1(adc)
+#define ADC_TR(adc)		ADC_TR1(adc)
+
+#define ADC1_ISR		ADC_ISR(ADC1)
+#define ADC1_IER		ADC_IER(ADC1)
+#define ADC1_CR			ADC_CR(ADC1)
+#define ADC1_CFGR1		ADC_CFGR1(ADC1)
+#define ADC1_CFGR2		ADC_CFGR2(ADC1)
+#define ADC1_SMPR		ADC_SMPR(ADC1)
+#define ADC1_TR			ADC_TR(ADC1)
+#define ADC1_CHSELR		ADC_CHSELR(ADC1)
+#define ADC1_DR			ADC_DR(ADC1)
+#define ADC1_CCR		ADC_CCR(ADC1)
+
+#define ADC1_CALFACT		ADC_CALFACT(ADC1)
+
```

Comment:
Here, these `ADC1_*` shortcut macros got introduced, and it seems like @karlp wanted them away.

I do like have some very small functions/macros just to make sure I did not make a fatal typo, but that is not the style of the house it seems. :)